src/Command/BaseCommand.php

```diff
@@ -15,28 +15,40 @@
 
 use Sonata\Doctrine\Model\ManagerInterface;
 use Sonata\MediaBundle\Provider\Pool;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 
 /**
  * This command can be used to re-generate the thumbnails for all uploaded medias.
  *
  * Useful if you have existing media content and added new formats.
  */
-abstract class BaseCommand extends ContainerAwareCommand
+abstract class BaseCommand extends Command
```

Comment:
IMHO we should deprecate this class, because it does not contain any logic at all

Reply:
Done